frontend/src/views/Home.js

```diff
@@ -1,29 +1,47 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
 import Button from '../components/Button/Button';
 import ViewWrappers from "../components/ViewWrappers/ViewWrappers";
 import Map from "../components/Map/Map";
+import { useLocationContext } from '../context/useCurrentLocation';
 
 import './Home.scss';
 
-const Home = () => (
-  <ViewWrappers.View>
-    <ViewWrappers.ViewSplitter small>
-      <div className="Home">
-        <div className="Intro">
-          <h1>Support your favourite local store online.</h1>
-          <p>Covexit enables small businesses to create an easy-to-use online shop to be able to master the corona crisis.</p>
-        </div>
+const Home = () => {
+  const { setCurrentLocation, coordinates } = useLocationContext()
+  const [requestLocation, setRequestLocation] = useState(false)
+  const { push } = useHistory();
+
+  const handleGeoLocation = (e) => {
+    e.preventDefault();
+    setCurrentLocation();
+    setRequestLocation(true);
+  }
```

Comment:
Can we do all of that inside <Map /> component? So we don't need to have all the logic inside <Home> and <StoreList>

Reply:
I think the new techniques is better